index.html

```diff
@@ -1,33 +1,46 @@
 ---
 layout: default
-desc: Netplan.io
+desc: netplan.io
 sitemap:
     priority: 1.0
     changefreq: 'monthly'
     lastmod: 2017-01-13T17:20:30+00:00
 ---
 
-<div id="main-content" class="p-strip is-bordered is-deep">
-  <div class="row">
+<div id="main-content" class="p-strip--hero is-bordered is-deep">
```

Comment:
Is there such a thing as `p-strip--hero`?

Reply:
removed